user_tools/src/spark_rapids_pytools/wrappers/databricks_aws_wrapper.py

```diff
@@ -92,6 +93,10 @@ def qualification(cpu_cluster: str = None,
                 "MATCH": keep GPU cluster same number of nodes as CPU cluster;
                 "CLUSTER": recommend optimal GPU cluster by cost for entire cluster;
                 "JOB": recommend optimal GPU cluster by cost per job.
+        :param estimation_model: Model used to calculate the estimated GPU duration and cost savings.
+               It accepts one of the following:
+               "XGBOOST": an XGBoost model for GPU duration estimation
```

Comment:
minor preference to use lowercase for argument values: XGBOOST --> xgboost

Reply:
Thanks @mattahrens !

The CLI handles both lower/upper-cases. I changed the comments to lower-case which reflects on the output of the `--help` command.